ui/src/app/pages/data/data-set/export-dataset-modal.tsx

```diff
@@ -61,9 +57,11 @@ const styles = reactStyles({
   },
 });
 
-export const ExportDatasetModal: (props: Props) => JSX.Element = fp.flow(
-  withCurrentWorkspace()
```

Comment:
This was the problem: there's no "current workspace" for resources on the homepage.  Now the workspace is passed explicitly, and most other changes in the PR are in support of that.